tests/lib/rule-tester/rule-tester.js

```diff
@@ -2295,6 +2228,123 @@ describe("RuleTester", () => {
         });
     });
 
+    describe("deprecations", () => {
+        let spy;
+
+        beforeEach(() => {
+            spy = sinon.spy(process, "emitWarning");
```

Comment:
spy will call the original `process.emitWarning` so we'll get warning logs on `npm test`. Can we use something else that doesn't call the original function, for example `sinon.stub`?

Reply:
`sinon.stub` works 👍🏻